core/sdk-core/src/main/java/software/amazon/awssdk/core/RequestOverrideConfiguration.java

```diff
@@ -103,9 +106,9 @@ public List<ApiName> apiNames() {
      * unmarshalling, etc. This value should always be positive, if present.
      *
      * <p>The api call timeout feature doesn't have strict guarantees on how quickly a request is aborted when the
-     * timeout is breached. The typical case aborts the request within a few milliseconds but there may occasionally be
-     * requests that don't get aborted until several seconds after the timer has been breached. Because of this, the client
-     * execution timeout feature should not be used when absolute precision is needed.
+     * timeout is breached. The typical case aborts the request within a few milliseconds but there may occasionally be requests
```

Comment:
Can we please revert unwanted changes  ? (Generic comment)
This becomes helpful when we are doing a final merge.

Reply:
Yeah, these are actually some indendation changes that I felt should go in for the sake of better readability and uniformity. Do we want to keep them as they are ?

Reply:
Lets keep them as they are